pset-functions.js

```diff
@@ -76,6 +91,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+  scores = { 'A' : 1, 'B' : 3, 'C' : 3, 'D' : 2, 'E' : 1, 'F' : 4, 'G' : 2, 'H' : 4, 'I' : 1, 'J' : 8, 'K' : 5, 'L' : 1, 'M' : 3, 'N' : 1, 'O' : 1, 'P' : 3, 'Q' : 10, 'R' : 1, 'S' : 1, 'T' : 1, 'U' : 1, 'V' : 4, 'W' : 4, 'X' : 4, 'Y' : 4, 'Z' : 10 };
+function scrabbleScore(word) {
+    word = word.toUpperCase();
+  let sum = 0;
+  for (i = 0; i < word.length; ++i) {
+      sum += scores[word.charAt(i)] || 0;
```

Comment:
Great use of a for loop and taking advantage of its iterator as an array's index. You are also taking edge cases into consideration by making sure each string character is uppercase. Tip: In line 99, another way of writing "scores[word.charAt(i)]" is "scores[word[i]]"